torch/onnx/_internal/fx/exporter.py

```diff
@@ -168,6 +168,12 @@ def aten_getitem(self, i):
     "aten::sum": ops.core.aten_sum_dim_IntList,
     "aten::argmin": ops.core.aten_argmin,
     "aten::argmax": ops.core.aten_argmax,
+    "aten::alias": ops.core.aten_alias,
+    "aten::cumsum": ops.core.aten_cumsum,
+    "aten::bitwise_not": ops.core.aten_bitwise_not,
+    "aten::select": ops.core.aten_select,
+    "aten::baddbmm": ops.core.aten_baddbmm,
+    "aten::masked_fill": ops.core.aten_masked_fill,
```

Comment:
Possible to sort the lines?

Reply:
I will close this and stack it on top of [validation PR](https://github.com/pytorch/pytorch/pull/94721)

Reply:
https://github.com/pytorch/pytorch/pull/94721 is unlikely to be merged soon.

Reply:
> Possible to sort the lines?

The whole dict wasn't sorted. Do you want me to sort it all?

Reply:
that'd be great thanks